challengeA/ishiwata/lib/energycharge.rb

```diff
@@ -0,0 +1,49 @@
+
+class EnergyCharge
+  attr_reader :usage
+
+  PLAN = ["従量電灯B", "おうちプラン", "ずっとも電気1"]
+  CHARGE = ["tokyo_energy", "looop_energy", "tokyogas_energy"]
+
+  def initialize(usage)
+    @usage = usage
+  end
+
+  def choose_plan(plan_name)
+    plan = PLAN.index(plan_name)
+    self.send(CHARGE[plan])
+  end
+
+
+  def tokyo_energy
+    energy_charge =
+      if usage > 300
+        30.57 * (usage - 300) + 26.48 * 180 + 19.88 * 120
+      elsif usage > 120
+        26.48 * (usage - 120) + 19.88 * 120
+      else
+        19.88 * usage
+      end
```

Comment:
何度も同じ定数を使うのは冗長だし、変更するとき、複数個所の修正が必要になる。
同じ処理はできる限りまとめる。

```suggestion
energy_charge = [
  19.88 * [ usage, 120 ].min,
  26.48 * [ [ usage, 300 ].min - 120, 0 ].max,
  30.57 * [ usage - 300, 0].max 
].sum
```